vendored/sqlite3-parser/src/parser/ast/mod.rs

```diff
@@ -534,6 +679,27 @@ impl Literal {
             unreachable!()
         }
     }
+
+    /// checks if two literal values are equivalent
+    fn is_equivalent(&self, other: &Self) -> bool {
+        match (self, other) {
+            (Self::Numeric(n1), Self::Numeric(n2)) => {
+                match (normalize_numeric_str(n1), normalize_numeric_str(n2)) {
+                    (Some(canonical), Some(canonical2)) => canonical == canonical2,
+                    _ => false,
+                }
+            }
+            // TODO: check for quoted == unquoted strings?
+            (Self::String(s1), Self::String(s2)) => s1.eq_ignore_ascii_case(s2),
```

Comment:
check this PR out: https://github.com/tursodatabase/limbo/pull/492

would be great to handle the same cases for equivalency, although ideally without extra allocations

Reply:
actually that PR was only for identifiers... so we could use the same logic for Id and Qualified

Reply:
All set :+1: